babel/core.py

```diff
@@ -1363,3 +1366,19 @@ def get_locale_identifier(
     lang, territory, script, variant, modifier = tup + (None,) * (5 - len(tup))
     ret = sep.join(filter(None, (lang, script, territory, variant)))
     return f'{ret}@{modifier}' if modifier else ret
+
+
+def get_cldr_version() -> str:
+    """Return the Unicode CLDR version used by this Babel installation.
+
+    Generally, you should be able to assume that the return value of this
```

Comment:
What other values can it have?

Reply:
Honestly, I'm pre-planning for a downstream distro to do something weird here like `"27+deb1"`. 😀

Reply:
Haha okay. My original request was to be able to link to version-specific CLDR docs so as long as there is some way to reliably get just the `27` part (either a separate function or by parsing a more complex value) that would support my use case :)

Happy holidays! 🎄😊